$MirOS$

	Want me to kill the person who wrote this now or later?

--- configure.in.orig	Wed Nov  1 16:33:45 2006
+++ configure.in	Fri Nov 24 17:37:49 2006
@@ -4,21 +4,8 @@
 AC_INIT(ocs-version)
 
 # -------------------------------------------------------------------------
-# Clearing unwanted defaults
-
-CC=
-CPP=
-CFLAGS=
-CPPFLAGS=
-LDFLAGS=
-LIBS=
-DEFS=
-
-# -------------------------------------------------------------------------
 # Check for C compiler
 
-CFLAGS=
-
 AC_PROG_CC 
 
 # Check for POSIX ISC
@@ -713,9 +700,9 @@ then
     then
       AC_MSG_WARN(cannot figure out how to link shared objects (ignored))
     fi
-    ocs_DLD="$ac_cv_path_CC $ocs_LDFLAGS $ocs_cv_flag_cc_shared"
+    ocs_DLD="$ac_cv_path_CC $LDFLAGS $ocs_cv_flag_cc_shared"
   else
-    ocs_DLD="$ac_cv_path_CC $ocs_LDFLAGS $ocs_cv_with_dldflags"
+    ocs_DLD="$ac_cv_path_CC $LDFLAGS $ocs_cv_with_dldflags"
   fi
 
   if test "$ocs_cv_with_dldlibs" != no
@@ -790,10 +777,10 @@ then
     AC_MSG_ERROR(cannot find out what flags your linker needs.)
   fi 
 
-  ocs_LD="$ocs_CC $ocs_cv_with_ldrpath\${OCSLIBPATH} $ocs_cv_with_ldrpathlink\${OCSLIBPATH}"
+  ocs_LD="$ocs_CC $LDFLAGS $ocs_cv_with_ldrpath\${OCSLIBPATH} $ocs_cv_with_ldrpathlink\${OCSLIBPATH}"
 else
   dnl no dynamic linker
-  ocs_LD=$ocs_CC        
+  ocs_LD="$ocs_CC $LDFLAGS"
 fi
 
 AC_CACHE_SAVE
